pkg/resources/cruisecontrol/configmap.go

```diff
@@ -42,7 +42,7 @@ import (
 
 const MinLogDirSizeInMB = int64(1)
 
-func (r *Reconciler) configMap(clientPass, capacityConfig string, log logr.Logger) runtime.Object {
+func (r *Reconciler) configMap(clientPass string, capacityConfig string, log logr.Logger) runtime.Object {
```

Comment:
I'm not sure why this needed to change. It should be fine as `varName, varName2 type` as it was originally shouldn't it?

Reply:
Yeah, it is a convention to shorten the function signature by omitting the continuously recurring type, but it does have a clarity reason to have the types stated explicitly, I would say it depends on personal taste.

Reply:
It has been changed because I put another variable between them but later I removed it.